src/TabWidget.js

```diff
@@ -0,0 +1,279 @@
+/**
+ * @class argos.TabWidget
+ * @alternateClassName TabWidget
+ */
+import declare from 'dojo/_base/declare';
+import lang from 'dojo/_base/lang';
+import array from 'dojo/_base/array';
+import domConstruct from 'dojo/dom-construct';
+import query from 'dojo/query';
+import _Templated from 'argos/_Templated';
+
+const __class = declare('argos.TabWidget', [_Templated], {
+  /**
+   * @property {Simplate}
+   * HTML that starts a new tab list
+   */
+  tabContentTemplate: new Simplate([
+    '<div class="panel-content" data-dojo-attach-point="contentNode">',
```

Comment:
The TabWidget doesn't use the contentNode. Move it back into _Detailbase. This will make the TabWidget more portal when attempting to use it in other views other than detail.